### DISCUSSION TITLE
The meat source problem

### DISCUSSION THREAD

**Discussion by Nick222** [widelands/wl_addons_server#110] [Q&A] [answered]
Hello!

Tell me, please, where does meat come from in the economic system of Europeans?

I understand that the hunter hunts animals - but where do the animals come from?

And is there any kind of meat farm in this system - like raising pigs, etc.?

Thanks :)

**Comment by Noordfrees** [comment]
@MarkMcWire this question is for you :)

**Comment by MarkMcWire** [comment, marked as answer]
If you enhance the basic hunter to normal and advanced hunter, the hunter has a gamekeeper function. So he creates animals and hunt them alternating.
Same for fisher, a normal and advanced fisher are fishbreeder, too.

Originally there was a pig farm, but I removed that because it didn't make any sense.
Instead, advanced fishermen are fishbreeders and advanced hunters are gamekeepers at the same time.
